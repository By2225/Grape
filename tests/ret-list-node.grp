fun Int main() {
    Graph<Int, Int> a;
    a = give_graph();
}

fun Graph<Int, Int> give_graph(){
    return << '5' -6- '7' >>;
}
