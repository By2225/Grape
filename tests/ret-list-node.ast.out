
fun Graph<Node<Int>, Edge<Int->Int>> give_graph() {
return <<'5' -6- '7' >>;
}

fun Int main() {
Graph<Node<Int>, Edge<Int->Int>> a;
a = give_graph();
}
